metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/MetacatConnectorProperties.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.common.server.util;
+
+import com.netflix.metacat.common.server.properties.Config;
+import com.netflix.spectator.api.Registry;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Server Context.
+ */
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+@Component
+public class MetacatConnectorProperties {
```

Comment:
1. Rename this class to something like `ConnectorConfig`. Metacat is redundant and properties is specific as we also pass in other objects and such
2. This is a DTO and as such should be completely immutable after construction. Please make all members `private final` and either provide an all args constructor or a builder to create the object. Anything required can't be nullable so if you use the builder pattern the builder constructor should take those fields as parameters.